packages/ui-react/src/components/VideoDetails/VideoDetails.module.scss

```diff
@@ -141,7 +141,7 @@
 .poster {
   position: absolute;
   top: 0;
-  right: -15px;
+  right: 0;
```

Comment:
This reverts a fix that @MelissaDTH did to hide the gap when removing the scroll. Are we sure this is the right fix?

![image](https://github.com/Videodock/ott-web-app/assets/3996119/191aaacc-2fb0-4215-8648-5c6788a9c208)

Reply:
We spoke about this on Slack, it's the best we can get at this moment. Perhaps a prettier scrollbar/overlay manager later.